client/ayon_maya/plugins/create/create_model.py

```diff
@@ -39,5 +39,9 @@ def get_instance_attr_defs(self):
                     placeholder="attr1, attr2"),
             TextDef("attrPrefix",
                     label="Custom Attributes Prefix",
-                    placeholder="prefix1, prefix2")
+                    placeholder="prefix1, prefix2"),
+            BoolDef("include_shaders",
+                    label="Include Shaders",
+                    tooltip="Include shaders in the geometry export.",
+                    default=False),
```

Comment:
@LiborBatek looking at this code I have a feeling the setting in settings does not influence the default state here at all?

Reply:
It should take it from a class attribute `self.include_shaders` similar to this: https://github.com/ynput/ayon-maya/pull/69/files#diff-d3f80c6626121c1d853dcc382eb66d18761a2aad4b6d7028125857fa5e552befR30

Of course `include_shaders = False` should also be added as class attribute.

```suggestion
                    default=self.include_shaders),
```

Reply:
I agree - @nikshankarnoble I tried to fix it but I don't have permissions to push to your fork. Could you just make those changes? With that, it is good to be merged. Thanks!

Reply:
I was able to push. ✅

Reply:
Lovely - thanks very much both!